### ISSUE TITLE
Cannot run n_pairs loss for TensorFlow 2.1 Windows

### ISSUE THREAD

**Issue by henry090** [henry090/tfaddons#1]
Why I get the following while specifying the loss function "loss_npairs"?

```
Not implemented in TF 2.1 Windows
```

**Comment by henry090** [comment]
/remind me to close this issue in 4 weeks
